seg_lapa/config/dataset/lapa.yaml

```diff
@@ -1,6 +1,7 @@
 # @package _group_
+_target_: "config_parse.dataset_conf.LapaConf"
```

Comment:
We don't want to use hydra's instantiate anywhere in our code. It'll be replaced by the new config_parsing module.
```suggestion
```